files/open-in-dvm.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Name=Open in Disposable VM
+Comment=Open file in a Disposable VM
+TryExec=/usr/bin/qvm-open-in-vm
+Exec=/usr/bin/qvm-open-in-vm '@dispvm:sd-viewer' %f
```

Comment:
Requires in `/etc/qubes-rpc/policy/qubes.OpenInVM`: 

```
sd-devices @dispvm:sd-viewer allow
```

(in https://github.com/freedomofpress/securedrop-workstation/pull/472)